foundry.toml

```diff
@@ -15,12 +15,10 @@ fail_on_revert = true
 [profile.default.fmt]
 wrap_comments = true
 
-
```

Comment:
Just so you know, I added 2 empty lines between each profile because a profile can have many subsections (e.g. fuzz, invariant, etc) so it's easier to distinguish between profiles and profile sections but obv not necessary